docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -1,80 +1,32 @@
-## Import terms into Mondo for use in logical axioms
-
-These workflows for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
+_Last updated 27-Mar-2024_
 
-### Preferred Instructions
+## Import terms into Mondo for use in logical axioms
 
-_Added 2020-05-06_
+This workflow is for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
 
-1. Open the `src/ontology/imports/*_terms.txt`
-2. Add term to this file
+1. Fetch the latest changes from `master` in the "mondo" repo
```

Comment:
When adding a new HGNC ID for a specific ticket, should the import refresh be done within that ticket or as a separate ticket?

Reply:
I talked with Sabrina about this and in theory it can be done either way. Generally though, when someone needs to refresh the imports they ask on Slack if anyone else needs anything and the imports are refreshed in their own PR.